### ISSUE TITLE
Integration with React/NextJS: results in console warnings on every render

### ISSUE THREAD

**Issue by adamrparsons** [dequelabs/axe-core-npm#594]
Product: react

Expectation: NextJS Example from this repo should be able to run without excessive warnings about the import

Actual: Either in a newly set up NextJS environment, or using the one in this repo's examples folder, when running the application, you can observe this warning both in the stdout and in the console

<pre><code>
./node_modules/axe-core/axe.js
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted
</code></pre>

This has been observed before in #211 and has been resolved there. I've tried installing the latest axe-core as well as latest @axe-core/react, and this behavior remains the same

Motivation: I would love to use this with our projects but the rate at which that error outputs the console (its quite excessive), it's making it a hard sell. 

As a side note; the example repo is a bit out of date. Dependabot has bumped NextJS ahead to a breaking change and you need to modify the `next.config.js` in order to start the dev server. 

<pre><code>
axe-core version: 4.5.0
@axe-core/react: 4.4.5

- Node version:  16
- Platform: Mac
</code></pre>

**Comment by samscarsella** [comment]
Also running into this with 
```axe-core: 4.5.1
@axe-core/react: 4.5.1
node: 14.19.0```
